gadgets/trace_signal/test/integration/trace_signal_test.go

```diff
@@ -59,7 +59,7 @@ func TestTraceSignal(t *testing.T) {
 
 	testContainer := containerFactory.NewContainer(
 		containerName,
-		"while true; do sleep 3 & kill $!; done",
+		"while true; do sleep 1 && sleep 3 & kill $!; done",
```

Comment:
I see it still generates a lot of events while testing locally, the following seems to fix the issue:


```suggestion
		"while true; do sleep 1 ; sleep 3 & kill $!; done",
```

Reply:
Can a bash magician explain why it doesn't work :sweat_smile: